setup.py

```diff
@@ -17,6 +17,7 @@
     'triton==2.0.0',
     'torchmetrics[image]==0.11.4',
     'clean-fid',
+    'lpips',
```

Comment:
Can you pin a version here? Unlikely to change but might save us a headache in the future.